github/repos_projects.go

```diff
@@ -91,7 +95,7 @@ type ProjectOptions struct {
 
 // CreateProject creates a GitHub Project for the specified repository.
 //
-// GitHub API docs: https://developer.github.com/v3/repos/projects/#create-a-project
+// GitHub API docs: https://developer.github.com/v3/projects/#create-a-repository-project
 func (s *RepositoriesService) CreateProject(owner, repo string, projectOptions *ProjectOptions) (*Project, *Response, error) {
 	u := fmt.Sprintf("repos/%v/%v/projects", owner, repo)
```

Comment:
This one should stay with `RepositoriesService` too, in addition to `ListProjects`. All others should move to `ProjectsService`.